docs/api/scheduler.md

```diff
@@ -1027,7 +1027,7 @@ Fetch all groups usage given a Partition and displays general information about
 ## GroupTracker
 ### Get specific group usage tracking information
 
-Fetch specific group usage given a Partition and displays general information about the groups managed by YuniKorn.
+Fetch specific group usage given a Partition and displays general information about the groups managed by YuniKorn. In case group name contains any special characters, needs to be url escaped to avoid issues.
```

Comment:
Nit: "In case the queue name contains any special characters, it needs to be url escaped to avoid issues."